src/ErrorHandler.php

```diff
@@ -106,23 +106,43 @@ public function memoryReserveSize(int $size): void
     }
 
     /**
-     * Register this error handler.
+     * Register PHP exception and error handlers (make once) and enable this error handler.
      */
     public function register(): void
     {
+        $this->initializeHandlers();
+        $this->enabled = true;
+    }
+
+    /**
+     * Disable this error handler.
+     */
+    public function unregister(): void
+    {
+        $this->enabled = false;
```

Comment:
We should still allow unregistering the handler. It could be called manually for whatever reason.

Reply:
I'm allow reset memory reserve here, but we don't can unregister handler.

1) Shutdown function in PHP can not be unregistered.

2) `restore_error_handler()` and `restore_exception_handler()` restore previous handler. We don't guarantee that between call `register()` and `unregister()` no other handlers were registered.